libs/ojsonnet/Eval_jsonnet.ml

```diff
@@ -15,8 +15,8 @@
 open Common
 open Core_jsonnet
 module A = AST_jsonnet
-module V = Value_jsonnet
 module J = JSON
+open Value_jsonnet
```

Comment:
Please put back the module alias instead of open. It's not documented anywhere I think but we usually try to avoid to use open Xxx because it makes it harder when reading the code to see where functions or types are coming from. It's usually
ok to open Common (because we're supposed to know the 50 basic functions that are in there), and to also open the file defining the main data structure used in a file (e.g., in Naming_AST.ml we do 'open AST_generic' because we know that's the only data structure used).
But here, we manipulate 2 data structures, which actually have sometimes the same name, so it can be confusing to know what is what, so better to qualify with V.xx the produced value.